src/test/java/org/apache/commons/io/filefilter/SymbolicLinkFileFilterTest.java

```diff
@@ -17,21 +17,198 @@
 
 package org.apache.commons.io.filefilter;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
+import java.io.File;
+import java.io.IOException;
 import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.function.BiFunction;
 
 import org.apache.commons.io.file.PathUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Don't change the order of static imports, keep them first.

Reply:
Done. CheckStyles doesn't have a test for this? (I'm not too familiar with CheckStyles anymore. It's been years.)